tf_encrypted/keras/models/sequential_test.py

```diff
@@ -87,6 +88,41 @@ def test_clone_model(self):
       np.testing.assert_allclose(actual, expected, rtol=1e-2, atol=1e-4)
 
 
+  def test_conv_model(self):
+
+    num_classes = 10
+    input_shape = (1, 28, 28, 1)
+    input_data = np.random.normal(size=input_shape)
+
+    with tf.Session():
+      model = tf.keras.models.Sequential()
+
+      model.add(tf.keras.layers.Conv2D(10,
```

Comment:
this value of channels seems high for a test -- can't we test the same with just 2?

Reply:
good point, we should reduce it.